coq_platform_make.sh

```diff
@@ -42,10 +42,16 @@ source shell_scripts/sanitize_environment.sh
 # NOTE: If you comment some questions for a custom setup, please assign
 #       the variable set in the script a default value.
 
-source shell_scripts/ask_introduction.sh
-source shell_scripts/ask_parallel_build.sh
-source shell_scripts/ask_compcert_open_source.sh
-source shell_scripts/ask_vst.sh
+COQ_PLATFORM_EXTENT=f
+# source shell_scripts/ask_introduction.sh
+COQ_PLATFORM_PARALLEL=p
+COQ_PLATFORM_JOBS=4
+#source shell_scripts/ask_parallel_build.sh
+COQ_PLATFORM_COMPCERT=n
+#source shell_scripts/ask_compcert_open_source.sh
+COQ_PLATFORM_VST=n
+# source shell_scripts/ask_vst.sh
+COQ_PLATFORM_SWITCH=d
```

Comment:
FYI: you can pass these to the platform script, see PLATFORM_ARGS in .github/workflow/*.yaml

Reply:
Yep, I realized later :)